dc-os/Docker/HA/Dockerfile

```diff
@@ -1,24 +1,26 @@
 FROM docker.bintray.io/jfrog/artifactory-pro:6.5.3
 
-MAINTAINER jainishs@jfrog.com
+LABEL MAINTAINER jainishs@jfrog.com
 
 # We download all the usual JDBC drivers available so we can configure them at runtime
 RUN curl -L -o  /opt/jfrog/artifactory/tomcat/lib/mysql-connector-java-5.1.38.jar https://bintray.com/artifact/download/bintray/jcenter/mysql/mysql-connector-java/5.1.38/mysql-connector-java-5.1.38.jar
-RUN apt-get update && apt-get install -y dnsutils
+#RUN apt-get update && apt-get install -y dnsutils
 
 COPY run.sh /runArtifactory.sh
 
 # Dynamic configurations
-COPY files/plugins/inactiveServerCleaner.groovy /tmp/inactiveServerCleaner.groovy
-COPY server.xml /opt/jfrog/artifactory/tomcat/conf/server.xml
-COPY binarystore.xml /tmp/binarystore.xml
-COPY artifactory.config.xml /tmp/artifactory.config.xml
+COPY --chown=artifactory:artifactory files/plugins/inactiveServerCleaner.groovy /tmp/inactiveServerCleaner.groovy
+COPY --chown=artifactory:artifactory server.xml /opt/jfrog/artifactory/tomcat/conf/server.xml
+COPY --chown=artifactory:artifactory binarystore.xml /tmp/binarystore.xml
+COPY --chown=artifactory:artifactory artifactory.config.xml /tmp/artifactory.config.xml
 
 # Pre-Populated keys
-RUN mkdir -p /var/opt/jfrog/artifactory/access/etc/keys/
-RUN mkdir -p /var/opt/jfrog/artifactory/etc/security/
-ADD files/access/etc/keys/private.key /var/opt/jfrog/artifactory/access/etc/keys/private.key
-ADD files/access/etc/keys/root.crt /var/opt/jfrog/artifactory/access/etc/keys/root.crt
-ADD files/security/communication.key /var/opt/jfrog/artifactory/communication.key
+RUN mkdir -p /var/opt/jfrog/artifactory/access/etc/keys/ /var/opt/jfrog/artifactory/etc/security/
+COPY --chown=artifactory:artifactory files/access/etc/keys/private.key /var/opt/jfrog/artifactory/access/etc/keys/private.key
```

Comment:
This section concerns me. I don't really feel like pre-populating the same keys in everyone's HA artifactory is a good idea.

Reply:
@TJM We don't need this anymore

Reply:
I will remove them.